spynl/tests/conftest.py

```diff
@@ -55,6 +55,17 @@ def app(settings):
 
 
 @pytest.fixture
+def app_factory():
```

Comment:
Thing is, i removed __init__py from tests so now you can't import. I did that as PyTest recommends.
I also thought about using imports that might be bad idea when you want to be sure that each test is using it's own fixtures etc, you do not need if for example a test patched the `app` fixture where you import it in 2 or more files and you run the tests in parallel.